docs/Override.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Symbol Override
+order: 211
+section: components
+---
+
+```javascript
+var overrides = symbolInstance.overrides
+```
+
+A [Symbol](https://www.sketchapp.com/docs/symbols/) override. This component is not exposed, it is only returns when accessing the `overrides` of a [Symbol Instance](#symbol-instance).
```

Comment:
@mathieudutour should the comment here read "it is only **_returned_** when accessing" or "it only **_returns_** when accessing"